Source/JavaScriptCore/API/JSContext.mm

```diff
@@ -81,13 +73,12 @@ - (instancetype)initWithVirtualMachine:(JSVirtualMachine *)virtualMachine
         return nil;
 
     m_virtualMachine = virtualMachine;
-    m_context = JSGlobalContextCreateInGroup(getGroupFromVirtualMachine(virtualMachine), 0);
+    m_context = JSGlobalContextRetain(JSGlobalContextCreateInGroup(getGroupFromVirtualMachine(virtualMachine), 0));
```

Comment:
This looks wrong. A function with the word create in its name creates an object that is already retained. Retaining it a second time will cause a storage leak, unless we somehow remember to release it twice.